seed/seed.json

```diff
@@ -1551,6 +1551,37 @@
             },
             "name": "BraveAutoTranslateStudy"
         },
+        {
+            "experiments": [
+                {
+                    "feature_association": {
+                        "disable_feature": [
+                            "BraveEnableAutoTranslate"
+                        ]
+                    },
+                    "name": "AutoTranslateDisabled",
+                    "probability_weight": 25
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 75
+                }
+            ],
+            "filter": {
+                "channel": [
+                    "STABLE",
```

Comment:
Should be `Release` rather than `Stable`. Same issue with https://github.com/brave/brave-variations/pull/486 which caused issues with CI due to not expecting `Stable`.

Reply:
Resolved it is @atuchin-m is on PTO.